### DISCUSSION TITLE
I want to remove the window that appears when unzipping.

### DISCUSSION THREAD

**Discussion by Square8** [aonez/Keka#1382] [Q&A]
![스크린샷 2024-01-05 오후 2 18 54](https://github.com/aonez/Keka/assets/102610524/fdc95730-b96a-4b28-ae2d-c26e72c22591)

hello.

The window doesn't open when I unzip...

The window pops out when I compress...
Is there any way to stop it?

**Comment by aonez** [comment]
Hi @Square8. What method do you use to compress the files? Contextual menu? Dropping files to the Keka icon?

**Reply by Square8** [reply]
I right-clicked and used the Keka compression below.
